docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -643,6 +643,32 @@ but you can still access the original via ``hostvars``::
 This works for all overriden connection variables, like ``ansible_user``, ``ansible_port``, etc.
 
 
+.. _scp_protocol_error_filename:
+
+How do I fix 'protocol error: filename does not match request' when fetching a file?
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Newer releases of OpenSSH have a bug in the SCP client that can trigger this error on the Ansible controller when using SCP as the file transfer mechanism::
+
+    failed to transfer file to /tmp/ansible/file.txt\r\nprotocol error: filename does not match request
+
+In these releases, SCP tries to validate that the path of the file to fetch matches the requested path.
+The validation
+fails if the remote filename requires quotes to escape spaces or non-ascii characters in its path. To avoid this error:
+
+* Use SFTP instead of SCP by setting ``scp_if_ssh`` to ``smart`` (which tries SFTP first) or to ``False``. You can do this in one of four ways:
+    * Rely on the default setting, which is ``smart`` - this works if ``scp_if_ssh`` is not explicitly set anywhere
+    * Set a host variable in the ``all`` group (``ansible_scp_if_ssh=smart`` or ``ansible_scp_if_ssh=False``)
+    * Set an environment variable: ``ANSIBLE_SCP_IF_SSH=smart`` or ``ANSIBLE_SCP_IF_SSH=False``
```

Comment:
This can confuse people who thing they can do

```
- module:
  environment:
    ANSIBLE_SCP_IF_SSH: smart
```

This won't work as env vars need to be set when calling Ansible itself.